rskj-core/src/main/java/co/rsk/peg/ErpRedeemScriptBuilderCreationException.java

```diff
@@ -4,22 +4,21 @@
  * Exception to be thrown when attempting to create a Federation with invalid values
  * that could result in the 2wp to stop working as expected
  */
-public class FederationCreationException extends RuntimeException {
+public class ErpRedeemScriptBuilderCreationException extends RuntimeException {
```

Comment:
FederationCreationException probably makes more sense since these are all errors related to the creation of the federation. If not we could merge this with ScriptCreationException class

Reply:
you are right